### PR TITLE
rfc-10, Exception

### PR DESCRIPTION
**Pull request by piogor**

### CODE REVIEW

File: 10/RequestReply.md [hunk 1]
```diff
@@ -76,6 +76,15 @@ SHOULD include:
 
 #### Exception
 
+Certain conditions may prevent a Request to be handled properly.
+
+* If a Request cannot be handled properly, the Client SHALL be notified by receiving of a DevFailed or MultiDevFailed exception (see [9/DataTypes.md](/9/DataTypes.md#Exceptions)).
```
Comment (bourtemb) [inline 10/RequestReply.md:81]: Is there a definition of a MultiDevFailed exception somewhere?

File: 10/RequestReply.md [hunk 2]
```diff
@@ -76,6 +76,15 @@ SHOULD include:
...
+* The DevFailed exception MAY be generated by either a Client (API) or a Device Server.
+* Exceptions MAY be rised either by API code or by a user provided code.
+* The Request Reply Protocol SHALL provide mechanism of sending a DevFailed or MultiDevFailded exception from a Device Server to a Client.
+* If a Device Server throws a (Multi)DevFailed excepiton upon request processing, the exception SHALL be sent to the Client which sent the request.
```
Comment (bourtemb) [inline 10/RequestReply.md]:
```suggestion
* If a Device Server throws a (Multi)DevFailed exception upon request processing, the exception SHALL be sent to the Client which sent the request.
```

File: 10/RequestReply.md [hunk 3]
```diff
@@ -76,6 +76,15 @@ SHOULD include:
 
 #### Exception
 
+Certain conditions may prevent a Request to be handled properly.
+
+* If a Request cannot be handled properly, the Client SHALL be notified by receiving of a DevFailed or MultiDevFailed exception (see [9/DataTypes.md](/9/DataTypes.md#Exceptions)).
+* The DevFailed exception MAY be generated by either a Client (API) or a Device Server.
+* Exceptions MAY be rised either by API code or by a user provided code.
```
Comment (bourtemb) [inline 10/RequestReply.md]:
```suggestion
* Exceptions MAY be raised either by API code or by a user provided code.
```

File: 10/RequestReply.md (hunk 1 above)
Comment (piogor) [reply 10/RequestReply.md:81]: This is missing in RFC-9.

Comment (piogor) [reply 10/RequestReply.md:81]: I have added #115 to make sure it will be added

Comment (piogor) [reply 10/RequestReply.md:81]: There is #116 fixing #115 :-), @bourtemb or @hardion, please review and merge if you find it is ok.

**Pull request merged**